### PR TITLE
add intro written for reddit to the docs

### PR DESCRIPTION
**Pull request by inhumantsar**
😉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,16 @@ instance -- at close to no cost.<sup>1</sup>
...
 
-> ![Email alert](docs/images/screenshots/alert-email.png)
+* It provides an an API Gateway to create no-permission AWS IAM users and access keys for those users.
+* You sprinkle the access keys wherever you like, for example in proprietary code or private data stores.
+* If one of those sources gets breached, an attacker is likely to use the stolen key to see what they can do with it.
+* You will receive an alert via email, PagerDuty, etc. that someone attempted to use the key.
+
+It's powered by Terraform, Lambda, Python 3, API Gateway, and DynamoDB.
+
+While the product itself is free and open source, you'll of course incur some AWS operating costs. However, it's designed to be run as cheaply as possible. Being completely serverless, you shouldn't be spending more than $1-$5 per month.
```
Comment (khicks) [inline README.md:19]: Remove the cost mention since it's already addressed in the footnote.

File: README.md [hunk 2]
```diff
@@ -7,9 +7,16 @@ instance -- at close to no cost.<sup>1</sup>
 
 ![SpaceSiren mascot](docs/images/logo/spacesiren-banner-medium.png)
 
-![Token API screenshot](docs/images/screenshots/token-api.png)
+## How it works
 
-> ![Email alert](docs/images/screenshots/alert-email.png)
+* It provides an an API Gateway to create no-permission AWS IAM users and access keys for those users.
```
Comment (khicks) [inline README.md:12]: Remove the word "Gateway".

File: README.md [hunk 3]
```diff
@@ -7,9 +7,16 @@ instance -- at close to no cost.<sup>1</sup>
...
+* You sprinkle the access keys wherever you like, for example in proprietary code or private data stores.
+* If one of those sources gets breached, an attacker is likely to use the stolen key to see what they can do with it.
+* You will receive an alert via email, PagerDuty, etc. that someone attempted to use the key.
```
Comment (khicks) [inline README.md:15]: Remove "via email, PagerDuty, etc."

File: README.md [hunk 4]
```diff
@@ -7,9 +7,16 @@ instance -- at close to no cost.<sup>1</sup>
...
+
+It's powered by Terraform, Lambda, Python 3, API Gateway, and DynamoDB.
```
Comment (khicks) [inline README.md:17]: Replace "It's" with "SpaceSiren is"

File: README.md [hunk 5]
```diff
@@ -7,9 +7,16 @@ instance -- at close to no cost.<sup>1</sup>
 
 ![SpaceSiren mascot](docs/images/logo/spacesiren-banner-medium.png)
 
-![Token API screenshot](docs/images/screenshots/token-api.png)
+## How it works
```
Comment (khicks) [inline README.md:10]: Title Case

> How It Works

**Comment by inhumantsar** [comment]
done!

**Pull request merged**